### ISSUE TITLE
Incorrect launcher icons on API 23

### ISSUE THREAD

**Issue by yeltayev22** [usefulness/easylauncher-gradle-plugin#124]
Hi! Thanks for the lib. 
But I see that for my emulator with API 23 the ribbon added is goes beyond the area of the icon itself. Is there any workaround?
https://ibb.co/FxQQn8V

**Comment by mateuszkwiecinski** [comment]
Hey @yeltayev22 👋 
Unfortunately, that's how the plugin by desing decorates the icon on APIs below 26. The behavior comes from previous implementation: https://github.com/akaita/easylauncher-gradle-plugin were it simply starts drawing from the top of an image (you app icon rosource has transparent frame around the icon). 

I can't think of any workaround right now, but I'll try to make the behavior customizable. I can't promise anything though - modifying launcher icon on lower APIs is quite complex :/

**Comment by mateuszkwiecinski** [comment]
I'm happy to announce that I just released version `3.10.0` which allows to override default behavior using:
```
filters customRibbon(
    ...,
    drawingOptions: ["IgnoreTransparentPixels"],
)
```
which should produce described above icon appearance. If needed, I also added additional option:
`drawingOptions: ["IgnoreTransparentPixels", "AddExtraPadding"],` which should generate exactly the same ribbon as on adaptive icon foreground resource. 

I've added these extra flags as a temporary solution until I figure out how to expose all customizable properties of a ribbon, with a simpler and more discoverable api 🤷

I hope new plugin capabilities will suit your need 🤞
